### DISCUSSION TITLE
Automated Network Tests

### DISCUSSION THREAD

**Discussion by ghost** [dtn7/dtn7-rs#36] [Q&A] [answered]
I wanted to run some of the tests under `dtn7-rs/tests/clab`. Specifically, I am interested in the `scenario_basic/` test. I was following the documentation to install *coreemu-lab*, but I am not sure where to install clab. Should I install clab on my local machine or your docker instance? Also, would do you mean by _copy it to your PATH_? Lastly, how do I go about running the `scenraio_basic\` test.

**Comment by gh0st42** [comment, marked as answer]
It is all included here, the docker images are pulled by the scripts in this repository.
Just try `./run_scenario scenario_basic` and it should all work if your docker host machine has `ebtables` and `sch_netem` kernel modules loaded, which is usually the case on ubuntu systems.
